corehq/apps/reports/views.py

```diff
@@ -425,7 +425,8 @@ def hq_update_saved_export(req, domain):
     index = int(req.POST['index'])
     group_config = get_document_or_404(HQGroupExportConfiguration, domain, group_id)
     config, schema = group_config.all_exports[index]
-    rebuild_export_task.delay(group_id, index)
+    filter = SerializableFunction(instances)
```

Comment:
I still don't quite understand how/whether deleted stuff was previously getting filtered from saved exports, but I think that's ok / can wait till we have tests :)